lib/tbot/config/configtemplate_ssh_host_cert.go

```diff
@@ -128,6 +169,26 @@ func (c *TemplateSSHHostCert) Render(ctx context.Context, bot Bot, currentIdenti
 		return trace.Wrap(err)
 	}
 
+	userCAs, err := bot.GetCertAuthorities(ctx, types.UserCA)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	// get the local domain name, used to exclude trusted CAs
+	localAuthName, err := bot.Client().GetDomainName(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	exportedCAs, err := exportSSHUserCAs(userCAs, localAuthName)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if err := dest.Write(c.Prefix+sshHostCASuffix, []byte(exportedCAs)); err != nil {
```

Comment:
This writes the exported CAs in "authorized keys" format (without the `cert-authority ` prefix) to a `.pub` file. Is this the correct format? RFC 4716 (https://www.rfc-editor.org/rfc/rfc4716) defines the ssh key/cert file format - in particular, the body must be wrapped at 72 chars, and I don't think the URL-encoded comments are the correct representation either.

I haven't yet grokked how these files are meant to be consumed, so I may have this wrong.

Reply:
This is for usage with OpenSSH's `TrustedUserCAKeys` configuration setting, which as far as I can tell only accepts the authorized keys format. This is to essentially automate the export of the Teleport User CA as done manually in this step of the OpenSSH and Teleport guide: https://goteleport.com/docs/server-access/guides/openssh/#step-14-configure-sshd-to-trust-the-teleport-ca

Reply:
Yup, this code is directly derived `tctl`'s existing implementation (as documented in the link). It works with existing implementations (at least on my Ubuntu 22.04 VM) so I'm inclined to leave things be.